io.catenax.pcf/4.0.0/Pcf.ttl

```diff
@@ -0,0 +1,896 @@
+   samm:characteristic :ProductFootprintVersion ;
+   samm:exampleValue "0"^^xsd:positiveInteger .
+
+:created a samm:Property ;
+   samm:preferredName "Creation of the Product (Carbon) Footprint"@en ;
+   samm:description "Timestamp of the creation of the Product (Carbon) Footprint as specified in the Catena-X PCF Rulebook in accordance with the technical specifications for PCF Data Exchange (Version 2.0.0) from the WBCSD (World Business Council for Sustainable Development)/ PACT initiative."@en ;
+   samm:see <https://wbcsd.github.io/tr/2023/data-exchange-protocol-20230221/> ;
+   samm:characteristic samm-c:Timestamp ;
+   samm:exampleValue "2022-05-22T21:47:32Z"^^xsd:dateTime .
+
+:status a samm:Property ;
+   samm:preferredName "Status"@en ;
+   samm:description "Status indicator of a product (carbon) footprint as specified in the technical specifications for PCF Data Exchange (Version 2.0.0) from the WBCSD (World Business Council for Sustainable Development)/ PACT initiative. WBCSD specific extension, in Catena-X set to \"Active\" per default."@en ;
+   samm:see <https://wbcsd.github.io/tr/2023/data-exchange-protocol-20230221/> ;
+   samm:characteristic :PfStatusCharacteristic ;
+   samm:exampleValue "Active" .
+
+:validityPeriodStart a samm:Property ;
+   samm:preferredName "Validity Period Start"@en ;
+   samm:description "Start of interval during which the product (carbon) footprint is declared as valid as specified in the Catena-X PCF Rulebook in accordance with the technical specifications for PCF Data Exchange (Version 2.0.0) from the WBCSD (World Business Council for Sustainable Development)/ PACT initiative. If specified, the validity period start must be equal to or greater than the reference period end."@en ;
+   samm:see <https://wbcsd.github.io/tr/2023/data-exchange-protocol-20230221/> ;
+   samm:characteristic samm-c:Timestamp ;
+   samm:exampleValue "2022-01-01T00:00:01Z"^^xsd:dateTime .
+
+:validityPeriodEnd a samm:Property ;
+   samm:preferredName "Validity Period End"@en ;
+   samm:description "End of interval during which the product (carbon) footprint is declared as valid as specified in the Catena-X PCF Rulebook in accordance with the technical specifications for PCF Data Exchange (Version 2.0.0) from the WBCSD (World Business Council for Sustainable Development)/ PACT initiative."@en ;
+   samm:see <https://wbcsd.github.io/tr/2023/data-exchange-protocol-20230221/> ;
+   samm:characteristic samm-c:Timestamp ;
+   samm:exampleValue "2022-12-31T23:59:59Z"^^xsd:dateTime .
+
+:comment a samm:Property ;
+   samm:preferredName "Comment"@en ;
+   samm:description "Additional information and instructions related to the calculation of the product (carbon) footprint as specified in the Catena-X PCF Rulebook in accordance with the technical specifications for PCF Data Exchange (Version 2.0.0) from the WBCSD (World Business Council for Sustainable Development)/ PACT initiative."@en ;
```

Comment:
same here rephrase ref to catenax as example. Please also check the other element descriptions

Reply:
Added "for example" to the Catena-X reference